build.sbt

```diff
@@ -1,32 +1,40 @@
-import sbtcrossproject.crossProject
+lazy val scalaVersionsJS  = Seq("2.13.4", "2.12.12", "2.11.12")
+lazy val scalaVersionsJVM = "3.0.0-M2" +: scalaVersionsJS
+lazy val scalaVersionsSN  = Seq("2.11.12")
+
+lazy val scalaTestVersion = "3.2.3"
 
 lazy val commonSettings = Seq(
   organization := "org.rogach",
   name := "scallop",
   version := {
     val versionRegexp = """[0-9]+\.[0-9]+\.[0-9]+""".r
-    val libraryDependenciesString =
-      scala.io.Source.fromFile("README.md").getLines.filter(_.contains("libraryDependencies")).mkString
```

Comment:
Yeah, I guess I was doing garbage collection in the "ultimate" way here (https://groups.google.com/forum/message/raw?msg=comp.lang.ada/E9bNCvDQ12k/1tezW24ZxdAJ) :)

Do we still have no safe Scala one-liner to read the file contents?

Reply:
Not sure; IntelliJ warned me that the file might not be closed, so I thought I make it explicit. I think `Source` has also a special clean-up method.

Reply:
No criticism here - you did everything correctly, the file indeed wasn't closed.